### PR TITLE
add Zandronum packages to nix

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: pkgs/games/zandronum/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    cat > $out/bin/zandronum << EOF
+    #!/bin/sh
+
+    LD_LIBRARY_PATH=$out/share $out/share/zandronum "\$@"
+    EOF
+
+    chmod +x "$out/bin/zandronum"
+
+    find $out
```
Comment (jagajaga) [inline pkgs/games/zandronum/default.nix]: Why do we need it here?

Comment (Lassulus) [inline pkgs/games/zandronum/default.nix]: whups, still left from debugging.

**Comment by MP2E** [comment]
Thanks for creating this! I've had a half baked Zandronum nix expression that I've been sitting on for a month or two, but I hadn't gotten it ready for release.

Will self assign to see this merged, thanks again!

**Comment by Lassulus** [comment]
It would be really cool if zandronum and zandronum-bin would be compatible with each other. But I can't figure out how to build it the right way. I just get the network protocol outdated warning everytime I try to connect to an online server.

**Pull request closed without merging**